opensrp-anc/src/main/java/org/smartregister/anc/library/fragment/ContactJsonFormFragment.java

```diff
@@ -118,10 +119,11 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
 
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
-        if (super.onOptionsItemSelected(item)) {
-            return true;
+        if (item.getItemId() == R.id.action_save) {
+            if (getActivity() != null) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/OpenSRP/opensrp-client-anc/pullRequest?prid=4647348)